jsonparse.ps1

```diff
@@ -0,0 +1,83 @@
+$location=$args[0]
+$repo=$args[1]
+$name=$args[2]
+
+$location
+$repo
+$name
+$repo.Title
+$JSON = (Get-Content $location) -join "`n" | ConvertFrom-Json
+
+$base = "https://github.com/"
+if($repo.Title -eq "github")
+{
+  $base = "https://github.com/"
+}
+#Renaming the key value of file
+$URI = $JSON.runs
```

Comment:
1. This variable doesn't contain a URI, so it's misnamed. But...

2. You don't actually use this variable before you assign it again. Is this some leftover debugging code?